sync2jira/main.py

```diff
@@ -282,14 +292,14 @@ def initialize_recent(config):
 
         # Deal with the message
         log.debug("Handling %r %r", suffix, entry["topic"])
-        msg = entry["msg"]
-        handle_msg({"msg": msg}, suffix, config)
+        body = c.extract_message_body(entry)
+        handle_msg({"body": body}, suffix, config)
 
 
-def handle_msg(msg, suffix, config):
+def handle_msg(body, suffix, config):
```

Comment:
`handle_msg()` now receives the unwrapped message body, right?  So line 296 should just pass `body` as the first parameter, without the wrapping, similar to line 177, right?

Reply:
You're right! Fixing.